tests/rust/prefix.rs

```diff
@@ -14,7 +14,7 @@ pub enum AbsoluteFontWeight {
 pub extern "C" fn root(x: NamedLenArray, y: ValuedLenArray, z: AbsoluteFontWeight) { }
 
 #[no_mangle]
-pub const X: i64 = 42 << 42;
+pub const X: i64 = 22 << 22;
```

Comment:
We should probably just use larger types for the intermediate representations. This mostly works around the issue, but it's fine.